src/isomorphic/children/flattenChildren.js

```diff
@@ -18,11 +18,7 @@ var warning = require('fbjs/lib/warning');
 
 var ReactComponentTreeHook;
 
-if (
-  typeof process !== 'undefined' &&
-  process.env &&
-  process.env.NODE_ENV === 'test'
-) {
```

Comment:
should i just `var ReactComponentTreeHook = require('ReactComponentTreeHook')` at the top of the file? 

This `ReactComponentTreeHook` is later conditionally required in `__DEV__` block. What about it then? Should something be adjusted?